Sources/Sentry/SentryOutOfMemoryScopeObserver.m

```diff
@@ -75,15 +75,19 @@ - (void)switchFileHandle
 
 - (void)store:(NSData *)data
 {
-    [self.fileHandle seekToEndOfFile];
-    [self.fileHandle writeData:data];
-    [self.fileHandle writeData:[@"\n" dataUsingEncoding:NSASCIIStringEncoding]];
-
-    self.breadcrumbCounter += 1;
-
-    if (self.breadcrumbCounter >= self.maxBreadcrumbs) {
-        [self switchFileHandle];
-        self.breadcrumbCounter = 0;
+    @try {
+        [self.fileHandle seekToEndOfFile];
```

Comment:
Let’s capture the return value and put it in the error log if it fails, the extra contract might help later.